elixir/apps/domain/lib/domain/config.ex

```diff
@@ -115,16 +115,24 @@ defmodule Domain.Config do
 
   ## Feature flag helpers
 
+  defp feature_enabled?(feature) do
+    fetch_env!(:domain, :enabled_features) |> Keyword.fetch!(feature)
+  end
+
   def sign_up_enabled? do
```

Comment:
Do we need all those helper functions if we have a `feature_enabled?/1`? It will raise anyway but if we keep adding feature flags this module can get pretty long.

Reply:
I can take them out if you want.  The way I think about feature flags is that they should be short lived.  If we've got a feature flag that is permanent, then in my opinion that should just be a configuration option, not a feature flag.  That's why I wasn't worried about this list of helpers getting too long.  Also, with a dedicated helper function it makes it really easy to locate anything that's gated by this feature flag.

Let me know what you'd prefer.

Reply:
You can see all flags in `config.ex` so there is no point in being overly explicit around that and I just like to have less code whenever that is reasonable. But this system is something, I feel, we will change a lot so I don't think that it's that important right now - whatever you prefer.

Reply:
Since the feature flags are meant to be temporary anyway and I agree, I think the feature flag system will definitely evolve over time, I think I'll just leave it as-is, since everything is working.  Going forward, I won't add any new helper functions like these.

Reply:
@bmanifold I think we should keep adding them, adding new ones without following existing code would be really confusing for people reading code, so whatever we do - we should be consistent with it.

Reply:
That's fair.  Hopefully, we can finish these up before adding more and then once these are gone we can only use the `feature_enabled?` function.